src/RequestForm.js

```diff
@@ -1374,26 +1366,30 @@ class RequestForm extends React.Component {
                                         name="item.barcode"
                                         placeholder={placeholder}
                                         aria-label={<FormattedMessage id="ui-requests.item.barcode" />}
+                                        label={<FormattedMessage id="ui-requests.item.barcode" />}
```

Comment:
Please omit `aria-label` now that this field properly has `label`.

Reply:
Fixed